src/serviceorchestration/5_0_0/sysd/serviceorchestration_sysd.tex

```diff
@@ -197,44 +197,44 @@ \section{Services produced}
 \label{sec:services}
 
 \msubsection{service}{orchestration}
-The purpose of this service is to find information about providers that meet the requirements. It also provided subscription functionality to repeated orchestrations (orchestration push). The service is offered for both application and core/support systems. 
+The purpose of this service is to find information about providers that meet the requirements. It also provided subscription functionality to repeated orchestrations (orchestration push). The service is offered for both application and Core/Support systems. 
 
-\msubsection{service}{orchestration-push-management}
-The purpose of this service is to manage orchestration push subscriptions in bulk. It also allows to signal the Service Orchestration system to orchestrate to related subscribers. The service is offered for both application and core/support systems. 
+\msubsection{service}{orchestrationPushManagement}
+The purpose of this service is to manage orchestration push subscriptions in bulk. It also allows to signal the Service Orchestration system to orchestrate to related subscribers. The service is offered for both application and Core/Support systems. 
 
-\msubsection{service}{simple-store-management}
-Recommended service (and only if the Service Orchestration system is using the simple store strategy). The purpose of this service is to add, remove and query simple store orchestration rules in bulk. The service is offered for core/support systems.
+\msubsection{service}{simpleStoreManagement}
+Recommended service (and only if the ServiceOrchestration system is using the simple store strategy). The purpose of this service is to add, remove and query simple store orchestration rules in bulk. The service is offered for Core/Support systems.
```

Comment:
and only if the ServiceOrchestration system is using the simple store strategy => only for systems that are using the simple store strategy